geonode_mapstore_client/client/themes/geonode/less/_resources-catalog.less

```diff
@@ -87,4 +87,18 @@
         font-style: italic;
         position: relative;
     }
+}
+
+.ms-resources-panel-wrapper {
+    .ms-resources-filter {
+        .ms-filters-form {
+            .ms-filter-accordion {
+                .form-group {
+                    label {
+                        width: 100%;
+                    }
+                }
+            }
+        }
+    }
```

Comment:
Do we need this? It is something specific to the geonode client?

Reply:
Yes, it's specific to geonode, as we place the count on the far right